Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/DefaultRiskLevels.kt

```diff
@@ -216,16 +159,313 @@ class DefaultRiskLevels @Inject constructor() : RiskLevels {
             Timber.d("Risk level changed and notification sent. Current Risk level is ${riskLevel.raw}")
         }
         if (lastCalculatedScore.raw == RiskLevelConstants.INCREASED_RISK &&
-            riskLevel.raw == RiskLevelConstants.LOW_LEVEL_RISK) {
+            riskLevel.raw == RiskLevelConstants.LOW_LEVEL_RISK
+        ) {
             LocalData.isUserToBeNotifiedOfLoweredRiskLevel = true
 
             Timber.d("Risk level changed LocalData is updated. Current Risk level is ${riskLevel.raw}")
         }
         RiskLevelRepository.setRiskLevelScore(riskLevel)
     }
 
+    private fun dropDueToMinutesAtAttenuation(
+        exposureWindow: ExposureWindow,
+        attenuationFilters: List<RiskCalculationParametersOuterClass.MinutesAtAttenuationFilter>
+    ) =
+        attenuationFilters.any { attenuationFilter ->
+            // Get total seconds at attenuation in exposure window
+            val secondsAtAttenuation = exposureWindow.scanInstances
+                .filter { attenuationFilter.attenuationRange.inRange(it.typicalAttenuationDb) }
+                .fold(0) { acc, scanInstance -> acc + scanInstance.secondsSinceLastScan }
+
+            val minutesAtAttenuation = secondsAtAttenuation / 60
```

Comment:
`Range` is a double AFAICT, don't we want `minutesAtAttenuation` to also be float or double and not int?

e.g.
`val minutesAtAttenuation: Double = secondsAtAttenuation / 60f`

Reply:
i was not actually sure from the tech spec if this should be rounded down or not, test cases all passed with this implementation but i will ask @mlenkeit to make sure

Reply:
Technically, secondsAtAttenuation is always divisible by 60 (as per EN spec). Anyway, I would still recommend not to make any implicit data type conversions...

Reply:
done in #1656